AppAuth-JS/built/node_support/index.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork